doc/ug/getting_started_sw_testing.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Test Software
+---
+
+## Prerequisites
+
+_Make sure you followed the install instructions to [prepare the system]({{< relref "install_instructions#system-preparation" >}}) and to install the [software development tools]({{< relref "doc/ug/install_instructions#software-development" >}}) and [Verilator]({{< relref "install_instructions#verilator" >}})._
+
+## Overview
+
+The OpenTitan software tests fall into two main categories:
+
+* Unit tests
+* Smoke tests
+
+Unit tests are designed to run on the host so they are quick to build and run.
+Writing unit tests is a good way to provide high levels of test coverage for a library without significantly impacting the runtime of the continuous integration tests.
+Smoke tests on the other hand are designed to run on an emulated device and therefore require a more complicated setup procedure.
```

Comment:
```suggestion
Smoke tests on the other hand are designed to run on-device and therefore require a more complicated setup procedure.
```

These tests run on simulations or FPGA prototypes of the OpenTitan chip, but will (likely) also run on the taped out chip. It's the same RTL, and to a large extend the same behavior.